src/um/wingdi.rs

```diff
@@ -2010,23 +2045,41 @@ ENUM!{enum DISPLAYCONFIG_COLOR_ENCODING {
     DISPLAYCONFIG_COLOR_ENCODING_INTENSITY = 4,
     DISPLAYCONFIG_COLOR_ENCODING_FORCE_UINT32 = 0xFFFFFFFF,
 }}
+STRUCT!{struct DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO_u_s {
+    bitfield: UINT32,
+}}
+BITFIELD!{DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO_u_s bitfield: UINT32 [
+    advancedColorSupported set_advancedColorSupported[0..1],
+    advancedColorEnabled set_advancedColorEnabled[1..2],
+    reserved set_reserved[2..32],
+]}
+UNION2!{union DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO_u {
+    [u32; 1],
+    s s_mut: DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO_u_s,
+    value value_mut : UINT32,
+}}
 STRUCT!{struct DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO {
     header: DISPLAYCONFIG_DEVICE_INFO_HEADER,
-    value: UINT32,
+    u: DISPLAYCONFIG_GET_ADVANCED_COLOR_INFO_u,
```

Comment:
In this case, where it is a union with only two variants, where one is a bitfield, and the other is the same type as the bitfield, I prefer to skip having a union and simply have the bitfield applied directly to that field. The layout is the same, but it makes working with the type easier.